.travis.yml

```diff
@@ -45,4 +45,8 @@ deploy:
     branch: master
     tags: true
     python: '3.6'
+    # admittedly, this condition is a bit silly but we want the deploy to happen
+    # on exactly one build, not all py36 builds, and looking for django 1.11 is a cheap
+    # way to accomplish that.
+    condition: $DJANGO = '1.11'
```

Comment:
I'm confused, by default if you test 10 Python versions Travis will run the deploy step 10 times?

Reply:
Not exactly. The deploy will run in any build where *all* of the `on` conditions are met. Up until this change, that meant any time a build ran against a tagged commit on master and we were testing against Python 3.6 Travis would attempt to deploy. As hinted by the `matrix` section, we allow Travis's test matrix to multiply the Python version by the `DJANGO` env var to determine what builds to run for each push. In our case that meant there were 4 builds that met the conditions: Python 3.6 with Django 1.11, Django 2.0, Django 2.1, and Django at the HEAD of their master branch.

This change adds an additional custom condition (aptly named `condition`) where that we use to explicitly pick a version of Django (and thus a single build) where the deploy should run.